src/index.test.js

```diff
@@ -13,7 +13,10 @@ jest.mock('DirectUploadProvider', () =>
 )
 
 const mockCSRFHeader = { 'X-CSRF-Token': 'qwertyuiop' }
-jest.mock('csrfHeader', () => () => mockCSRFHeader)
+// jest.mock('csrfHeader', () => (headers) => {
```

Comment:
Since we don't need to test what's being sent to csrfHeader anymore, can you restore the simple mock and remove the DOM modification on L65? Everything should still pass.

Reply:
good point